<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Back to Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(35, 35, 35);
border-radius: 10px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="recommendationsButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>More Recommendations</string>
   </property>
  </widget>
  <widget class="QLabel" name="recdisplayLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>271</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(45, 45, 45);
border-radius: 10px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="movielistWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(35, 35, 35);
border-radius: 10px</string>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="reclistWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(45, 45, 45);
border-radius: 10px
</string>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="recheadLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(45, 45, 45);
border-radius: 10px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
